integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -104,6 +110,12 @@ public final class AlluxioJniFuseFileSystem extends AbstractFuseFileSystem
       = new IndexedSet<>(ID_INDEX, PATH_INDEX);
   private final AuthPolicy mAuthPolicy;
   private final FuseFileStream.Factory mStreamFactory;
+  /**
+   * Making sure only one write stream at a time.
+   * 256 locks since libfuse working threads (guarded by MAX_IDLE_THREADS)
+   * normally are less than 256. Each lock takes about 100 bytes, total takes around 30kb.
+   */
+  private final Striped<ReadWriteLock> mPathLocks = Striped.readWriteLock(256);
```

Comment:
So this is a lock pool? Should it be configurable number?

Reply:
I kind of do not want to keep adding new property keys, do you think users will use more than 256 concurrencies (libfuse threads)? 256 uses relatively little memory

Reply:
cannot get the hash collision probability calculation.
Normally fuse working threads = 64 or 128 threads. At most 128 locks are needed. Considering the hash collision, 128 * 2 = 256 is likely to be enough.

@beinan any idea on how many locks should be allocated and should i make it configurable?